src/services/broadcast-service.js

```diff
@@ -79,11 +79,11 @@ class BroadcastService {
   }
 
   broadcastMessage() {
+
     let sentBy = `\n\nBroadcast message sent by: ${this.userEmail}`;
     let messageToSend = this.message + sentBy;
     if (this.ackFlag) {
       messageToSend = `${messageToSend}\nPlease acknowledge this message by replying with /ack`;
-      sentBy = `${sentBy}\nPlease acknowledge this message by replying with /ack`;
```

Comment:
Did you replace this in the send voice memo and files functions?

Reply:
I did not! pushing a change now

Reply:
I don't seem to be running into any issue with sentby taken out , because messageToSend is there. but I do see where sentby is right after that definition, and where it should be messageToSend, so I think I should just replace the functions like sendSecurityGroupAttachment using sentby, to messageToSend.  sentby was causing a double acknowledgment message